<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdgeParams</class>
 <widget class="QWidget" name="EdgeParams">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>307</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Relation params</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../share/mainResources.qrc">
    <normaloff>:/textedit/images/editpaste.png</normaloff>:/textedit/images/editpaste.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="addWire">
       <property name="toolTip">
        <string>Add wire</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../share/mainResources.qrc">
         <normaloff>:/icons/images/package-new.png</normaloff>:/icons/images/package-new.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delWire">
       <property name="toolTip">
        <string>Del wire</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../share/mainResources.qrc">
         <normaloff>:/icons/images/package-remove.png</normaloff>:/icons/images/package-remove.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="wiresList">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../../share/mainResources.qrc">
         <normaloff>:/textedit/images/filesave.png</normaloff>:/textedit/images/filesave.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../share/mainResources.qrc">
         <normaloff>:/icons/images/new/gtk-no.png</normaloff>:/icons/images/new/gtk-no.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAdd_wire">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icons/images/package-new.png</normaloff>:/icons/images/package-new.png</iconset>
   </property>
   <property name="text">
    <string>Add wire</string>
   </property>
   <property name="shortcut">
    <string>+, Ctrl+A</string>
   </property>
  </action>
  <action name="actionRemove_wire">
   <property name="icon">
    <iconset resource="../../share/mainResources.qrc">
     <normaloff>:/icons/images/package-remove.png</normaloff>:/icons/images/package-remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove wire</string>
   </property>
   <property name="shortcut">
    <string>-, Del, Ctrl+D</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q, Esc</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S, Return</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../share/mainResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addWire</sender>
   <signal>pressed()</signal>
   <receiver>actionAdd_wire</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delWire</sender>
   <signal>pressed()</signal>
   <receiver>actionRemove_wire</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
